spec/lib/web-server.spec.js

```diff
@@ -342,33 +390,26 @@ describe("WebServer", () => {
                 expect(args[0]).toBe(8080);
                 expect(args[1]).toEqual(jasmine.any(Function));
                 expect(args.length).toBe(2);
-            }).then(done, done);
+            });
         });
-        it("has a working callback", (done) => {
-            var callback;
-
-            webServer.startServerAsync().then(() => {
-                callback = restifyServer.listen.mostRecentCall.args[1];
-                expect(() => {
-                    callback();
-                }).not.toThrow();
-            }).then(done, done);
+        it("has a working callback", () => {
```

Comment:
Correct.  The callback should simply not throw.  Throwing would reject the promise.